pkg/cloudprovider/providers/gce/gce_loadbalancer_external_test.go

```diff
@@ -330,7 +330,7 @@ func TestUpdateExternalLoadBalancer(t *testing.T) {
 	err = gce.updateExternalLoadBalancer("", svc, newNodes)
 	assert.NoError(t, err)
 
-	lbName := cloudprovider.GetLoadBalancerName(svc)
+	lbName := cloudprovider.DefaultGetLoadBalancerName(svc)
```

Comment:
Should be `gce.GetLoadBalancerName(ctx, clusterName, svc)`, same for the rest of the test case.